ci/Conformance.md

```diff
@@ -0,0 +1,67 @@
+# Conformance results
+
+## To check for the existing nodes
+```
+$ kubectl get nodes
```

Comment:
Why?

Reply:
Have you tried to copy and paste when you have to delete the first 2 characters of each line.  Without the leading '$' you can just double click and copy the entire text box.  Be it one line or 10.